src/js/plugins/plugin.search.js

```diff
@@ -531,11 +502,18 @@ BookReader.prototype.addSearchResult = function(queryString, pageIndex) {
     )
     .data({ pageIndex })
     .appendTo(this.$('.BRnavline'))
-    .bt(searchBtSettings)
     .hover(
       (event) => {
         // remove from other markers then turn on just for this
         // XXX should be done when nav slider moves
+        const marker = event.currentTarget;
+        const tooltip = marker.querySelector('.BRquery');
+        const tooltipOffset = tooltip.getBoundingClientRect();
+        const targetOffset = marker.getBoundingClientRect();
+        const boxSizeAdjust = parseInt(getComputedStyle(tooltip).paddingLeft) * 2;
+        if (tooltipOffset.x - boxSizeAdjust < 0) {
+          tooltip.style.setProperty('transform', `translateX(-${targetOffset.left - boxSizeAdjust}px)`);
```

Comment:
Q: does this fix also update the page results highlights z-index issue where it bleeds into the tooltip?

Reply:
What's the exact issue? I tested this with chapter markers and with search results, and both rendered fine. Can you tell me how to reproduce what you're describing?

Reply:
go here: https://archive.org/details/goodytwoshoes00cran/page/5/mode/2up?q=Margery

hover over the second to last or last search droplet - and you will see the page highlight bleed onto the tooltip highlight
![image](https://user-images.githubusercontent.com/7840857/89198197-b006c880-d561-11ea-9bb2-343ff21f0a0a.png)

Reply:
No, it still occurs because of a lack of z-index on `.BRcontainer`. I've added one to fix it.